test/utils/polling_client.test.js

```diff
@@ -208,7 +283,7 @@ describe('utils/polling_client', function () {
         expect(httpClient.post.callCount).to.equal(2);
         clock.tick(150000);
         expect(httpClient.post.callCount).to.equal(2);
-        clock.tick(490000);
+        clock.tick(487000);
```

Comment:
Had to update this test once I fixed the `pollingIntervalMs` option above. Before, it was using the default 5000ms interval.